src/components/GeneralInfo/SystemCard/SystemCard.test.js

```diff
@@ -1,27 +1,34 @@
-/* eslint-disable camelcase */
+import '@testing-library/jest-dom';
+import { screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import MockAdapter from 'axios-mock-adapter';
 import React from 'react';
-import toJson from 'enzyme-to-json';
-import SystemCard from './SystemCard';
+import { Provider } from 'react-redux';
+import { useParams } from 'react-router-dom';
 import configureStore from 'redux-mock-store';
-import { rhsmFacts, testProperties } from '../../../__mocks__/selectors';
 import promiseMiddleware from 'redux-promise-middleware';
-
-import { hosts } from '../../../api/api';
-import MockAdapter from 'axios-mock-adapter';
+import { renderWithRouter } from '../../../Utilities/TestingUtilities';
 import mockedData from '../../../__mocks__/mockedData.json';
-import { Provider } from 'react-redux';
-import { mountWithRouter } from '../../../Utilities/TestingUtilities';
-import { Clickable } from '../LoadingCard/LoadingCard';
-import { useParams } from 'react-router-dom';
+import { rhsmFacts, testProperties } from '../../../__mocks__/selectors';
+import { hosts } from '../../../api/api';
+import SystemCard from './SystemCard';
+
+const fields = [
+  'Host name',
+  'Display name',
+  'Ansible hostname',
+  'SAP',
+  'System purpose',
+  'Number of CPUs',
+  'Sockets',
+  'Cores per socket',
+  'CPU flags',
+  'RAM',
+];
+
 const mock = new MockAdapter(hosts.axios, { onNoMatch: 'throwException' });
 
 const location = { pathname: 'some-path' };
-const removeLabelledBy = ({
-  'aria-labelledby': labelledBy,
-  'aria-describedby': describedby,
-  id: id,
-  ...restProps
-}) => restProps;
 
 jest.mock('react-router-dom', () => ({
```

Comment:
Unless it needs a specific location or params, I think we can remove this mock, since it should work fine when rendering with a router instance.